cliquet/__init__.py

```diff
@@ -26,7 +26,7 @@
 DEFAULT_SETTINGS = {
     'cliquet.backoff': None,
     'cliquet.batch_max_requests': 25,
-    'cliquet.cache_backend': 'cliquet.cache.redis',
+    'cliquet.cache_backend': 'cliquet.cache.memory',
```

Comment:
Ok I get you.

IMO switching from Redis to Memory is still out of scope :) :hear_no_evil: